app/views/directives/deploy-image.html

```diff
@@ -167,8 +176,6 @@
             help-text="Each label is applied to each created resource.">
         </label-editor>
 
-        <alerts alerts="alerts"></alerts>
```

Comment:
Removing this alert cause we alreadty have `alerts` directive in view https://github.com/openshift/origin-web-console/blob/master/app/views/create.html#L13